lib/src/table/table-body/table-body.component.html

```diff
@@ -0,0 +1,7 @@
+<tr *ngFor="let row of rows; let i = index" (click)="handleClickRow({ item: row, index: i })">
+    <td *ngIf="selectable" class="table-checkbox">
+        <input type="checkbox" class="form-control" [checked]="selectedRows.includes(i)" />
+    </td>
+
+    <td seb-table-td *ngFor="let column of headerList" [row]="row[column.tableKeySelector]" [valueType]="column.valueType"></td>
```

Comment:
do we need headerList inside table-body components

Reply:
I think it's because the information for valueType is coming from the header list.